pkg/apis/events/v1alpha1/cloudstoragesource_conversion.go

```diff
@@ -27,7 +26,7 @@ import (
 
 // ConvertTo implements apis.Convertible.
 // Converts source (from v1beta1.CloudStorageSource) into v1alpha1.CloudStorageSource.
-func (source *CloudStorageSource) ConvertTo(_ context.Context, to apis.Convertible) error {
+func (source *CloudStorageSource) ConvertTo(ctx context.Context, to apis.Convertible) error {
```

Comment:
Just want to double check with one thing: when we synced offline you mentioned that we need to add conversion between v1 and v1alpha1. Here I found you only added UT for conversion between v1alpha1 and v1, is that enough? Like we only need to add UT for this conversion, but no need to support the logic that converts directly between v1alpha1 and v1. (Because when we synced offline, you just mentioned we need to add conversion between v1 and v1alpha1) just want to double check specifically.

Reply:
I'm kind of confused with your comment. https://github.com/capri-xiyue/knative-gcp/blob/1ed880e121c52579ac3d20be4b20dd079b1d35db/pkg/apis/events/v1alpha1/cloudstoragesource_conversion.go#L43 this line does conversion between v1alpha1 and v1 and I also added UT related to it

Reply:
This is what knative serving does, the conversion v1alpha1 -> v1 is through v1alpha1 -> v1beta1 -> v1 https://github.com/knative/serving/blob/e071715848e6e8677c16c2dab129e6d131a2f196/pkg/apis/serving/v1alpha1/route_conversion.go#L36

Reply:
My bad. I got it. Thanks for point it out!